pkgs/development/python-modules/aioconsole/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildPythonPackage, fetchzip
+, pytest, pytest-asyncio, pytestcov }:
+
+# This package provides a binary "apython" which sometimes invokes
+# [sys.executable, '-m', 'aioconsole'] as a subprocess. If apython is
+# run directly out of this derivation, it won't work, because
+# sys.executable will point to a Python binary that is not wrapped to
+# be able to find aioconsole.
+# However, apython will work fine when using python##.withPackages,
+# because with python##.withPackages the sys.executable is already
+# wrapped to be able to find aioconsole and any other packages.
+buildPythonPackage rec {
+  pname = "aioconsole";
+  version = "0.1.7";
+  name = "${pname}-${version}";
+
+  # tests are not included in the version on PyPI
+  src = fetchzip {
+    url = "${meta.homepage}/archive/v${version}.zip";
+    sha256 = "0p12viyngplqds7xkwysfh1nlgrny8hwja2mm2na0wzfxj04fbcd";
+  };
+
+  buildInputs = [ pytest pytest-asyncio pytestcov ];
```

Comment:
`checkInputs`

Also, you were building without sandbox and missed `pytestrunner`.